### PR TITLE
Propose Siamak's repos

### PR DESCRIPTION
**Pull request by siamak**

### CODE REVIEW

**Comment by mamal72** [comment]
Please rebase your PR. There are some conflicts.

**Comment by mohebifar** [comment]
We are going to keep boilerplates away from specifically JS, CSS or HTML. (See #2) I would appreciate it if you update your PR.

File: data.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
         "jalaali/jalaali-js",
         "jalaali/moment-jalaali",
         "mamal72/telegram-github-search-bot",
+<<<<<<< HEAD
```
Comment (mamal72) [inline data.json]: Not yet. 😁

**Comment by mohebifar** [comment]
Ah, it could not merge it automatically. PLEASE merge and squash your commits.

**Comment by mohebifar** [comment]
Please squash.

**Pull request closed without merging**